pkg/dockerregistry/client.go

```diff
@@ -22,7 +22,7 @@ import (
 // Client includes methods for accessing a Docker registry by name.
 type Client interface {
 	// Connect to a Docker registry by name. Pass "" for the Docker Hub
-	Connect(registry string, allowInsecure bool) (Connection, error)
+	Connect(registry string, allowInsecure, disableV2 bool) (Connection, error)
```

Comment:
So we can let users work around bugs for now where v1 would work with
fallback.

On Aug 11, 2015, at 11:30 AM, Andy Goldstein <notifications@github.com>
wrote:

In pkg/dockerregistry/client.go
<https://github.com/openshift/origin/pull/4104#discussion_r36758904>:

> @@ -22,7 +22,7 @@ import (
>  // Client includes methods for accessing a Docker registry by name.
>  type Client interface {
>  	// Connect to a Docker registry by name. Pass "" for the Docker Hub
> -	Connect(registry string, allowInsecure bool) (Connection, error)
> +	Connect(registry string, allowInsecure, disableV2 bool) (Connection, error)

Why have disableV2? All of Docker will be moving to v2 over the coming
months.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4104/files#r36758904>.